### PR TITLE
[IN-2191] Ruby gems in weekly

### PR DESCRIPTION
**Pull request by MrBlaise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [x] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [x] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [x] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

**Review by fehersanyi-bitrise — APPROVED** [review]
lgtm

File: roles/ruby-gems/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+- name: Install gems
+  import_tasks: gems.yml
```
Comment (bitrise-tom) [inline roles/ruby-gems/tasks/main.yml:3]: I would do away with this import and put the contents of gems.yml in main.yml

**Review by bitrise-tom — CHANGES_REQUESTED** [review]

File: roles/ruby-gems/tasks/main.yml (hunk 1 above)
Comment (MrBlaise) [reply roles/ruby-gems/tasks/main.yml]: 👍

**Comment by MrBlaise** [comment]
The website having difficulties (getting 500s) hence the not green github check but the tests are green

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Review by fehersanyi-bitrise — APPROVED** [review]

**Review by bitrise-tom — APPROVED** [review]

**Pull request merged**